src/searchbar.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+    import Typeahead from './Typeahead.vue'
+    export default {
+      extends: Typeahead,
+      computed: {
+        primitiveData() {
+          if (this.data) {
+            var matches = []
+            var words
+            this.data.map(value => {
+              var query = new RegExp(this.value, 'i')
```

Comment:
if the variable does not change its value after initialization, you should use `const` instead of `var`:

```js
const query = new RegExp(this.value, 'i')
```

Same for the rest of the variables below.